### ISSUE TITLE
Theme broken with One Light UI Theme 0.44.0 & Atom 1.10.0

### ISSUE THREAD

**Issue by gitkrax** [nylki/pen-paper-coffee-syntax#28]
For some reason the theme is broken with the One Light UI Theme 0.44.0 and Atom 1.10.0. The problems disappear when changing to a different UI theme, e.g. Atom Light. See images for examples:

![sieppaa](https://cloud.githubusercontent.com/assets/5079912/18246878/59df1674-7377-11e6-83bf-8e38fa56a211.PNG)

![sieppaa](https://cloud.githubusercontent.com/assets/5079912/18246898/800ff872-7377-11e6-9f25-583eed40a544.PNG)

There are many other instances with similar problems and other problems, i.e. not recognizing italics, only some of the citations etc.

**Comment by nylki** [comment]
Hey @gitkrax,
This looks unusual indeed! 

I am currently also using the theme with One Light UI 0.44.0 and Atom 1.10.0, but don't experience any of those issues.

Could you please check wether the correct grammar is selected in the bottom right of the editor (eg. `Github Markdown` or `Markdown`)? 

![bildschirmfoto von 2016-09-05 15-36-43](https://cloud.githubusercontent.com/assets/1710598/18249526/27bd96fe-737f-11e6-86d4-81e92d093997.png)
![bildschirmfoto von 2016-09-05 15-36-00](https://cloud.githubusercontent.com/assets/1710598/18249527/27c1297c-737f-11e6-815d-de3c87e57f54.png)


Furthermore, please check what happens if you select a default syntax theme.  Does the issue remain?

If that doesn't help: Did you install any other packages recently that could cause the issue?

**Comment by gitkrax** [comment]
I disabled all other packages, tried other themes, and the problem seems specific to this theme.

To answer your questions:

* I did have the grammar set to Pandoc Markdown, but the problem persists with GitHub Markdown.
* The default syntax theme does not have this problem.
* Restarting Atom does not help.
* I disabled all installed packages and the problem persisted. I then removed all packages that I have installed within a month and the problem still persisted.
